tests/catchup_subscription_handles_small_batch_sizes.php

```diff
@@ -24,14 +24,10 @@
 use Prooph\EventStoreClient\ExpectedVersion;
 use Prooph\EventStoreClient\Internal\ResolvedEvent;
 use ProophTest\EventStoreClient\Helper\Connection;
-use Throwable;
```

Comment:
Sometimes you import `Throwable`, sometimes not. Maybe it should be consistent over all classes.

Reply:
Yes, definitely. PHPStorm prefers to add the `\` instead of important classes vom global namespace. I tried to clean that up already multiple times, but each new change PHPStorm again... :-)

Reply:
You can configure PhpStorm to prefer either FQN or import for classes from global namespace: 
Preferences -> Editor -> Auto Import -> [Treat symbols from the global space]

Reply:
Thanks, I'll give it a try.